utils/streams.js

```diff
@@ -0,0 +1,147 @@
+import fs from "fs";
+import path from "path";
+import https from "https";
+import { promisify } from "util";
+import minimist from "minimist";
+import through2 from "through2";
+import colors from 'colors'; // just for fun =)
+import config from "./../config";
+import { csvToJSON } from './';
+
+// task 4
+const extractFileDataNames = ["extract", "extractFileData", "printFileData", "getFileData"];
```

Comment:
What is the reason of defining such arrays (the same for others below)?

Reply:
@DraXu3 The reason is that according to tasks we may maintain the calling of these functions by different names, e.g. in case of ```inputOutput``` it should be possible to call it like _io_, _inputOutput_, _input-output_